docs/headless-chrome.md

```diff
@@ -75,7 +75,7 @@ const lighthouse = require('lighthouse');
 const chromeLauncher = require('lighthouse/chrome-launcher');
 
 function launchChromeAndRunLighthouse(url, flags = {}, config = null) {
-  return chromeLauncher.launch().then(chrome => {
+  return chromeLauncher.launch(flags).then(chrome => {
```

Comment:
hate to bikeshed a small commit like this but...

should we call this `options` instead of `flags`? 

also we're technically providing the same flags to both LH and CLauncher, even tho launcher has [unique options](https://github.com/GoogleChrome/lighthouse/tree/master/chrome-launcher#launchopts). it just so happens that `chromeFlags` and `port` are the same in both.
so a slightly more correct example would have different options objects for both launcher and LH. :/
wdyt?